pkg/deploy/controller/deploymentconfig/deploymentconfig_controller.go

```diff
@@ -84,11 +89,29 @@ func (c *DeploymentConfigController) Handle(config *deployapi.DeploymentConfig)
 		return c.updateStatus(config, []*kapi.ReplicationController{})
 	}
 
-	// Find all deployments owned by the deployment config.
+	// List all ReplicationControllers to find also those we own but that no longer match our selector.
+	// They will be orphaned by ClaimReplicationControllers().
+	rcList, err := c.rcLister.ReplicationControllers(config.Namespace).List(labels.Everything())
+	if err != nil {
+		return fmt.Errorf("DeploymentConfigController can't list all RCs: %s", err)
```

Comment:
also the format we use is:

`"error while deploymentConfigController listing the replication controllers: %v"`